app/code/core/Mage/Api/Model/Server/Wsi/Handler.php

```diff
@@ -31,7 +31,7 @@
  * @package    Mage_Api
  * @author     Magento Core Team <core@magentocommerce.com>
  */
-class Mage_Api_Model_Server_WSI_Handler extends Mage_Api_Model_Server_Handler_Abstract
+class Mage_Api_Model_Server_Wsi_Handler extends Mage_Api_Model_Server_Handler_Abstract
```

Comment:
Code Change: Class renamed

Reply:
IMHO a bug that possibly causes autoloader to fail ... please try `$test = new Mage_Api_Model_Server_WSI_Handler();` on unchanged code ....

=> `Fatal error: Uncaught Error: Class 'Mage_Api_Model_Server_WSI_Handler' not found`

Reply:
I use openmage-lts 1.9.3.x, installed with composer.
Used magerun dev:console

Both worked: 
>>> $test = new Mage_Api_Model_Server_WSI_Handler();
=> Mage_Api_Model_Server_WSI_Handler {#1162}
>>> $test = new Mage_Api_Model_Server_Wsi_Handler();
=> Mage_Api_Model_Server_WSI_Handler {#1114}


```
 php -v
PHP 7.1.28 (cli) (built: Apr 19 2019 00:35:14) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2018 Zend Technologies
    with Xdebug v2.7.1, Copyright (c) 2002-2019, by Derick Rethans
    with Zend OPcache v7.1.28, Copyright (c) 1999-2018, by Zend Technologies
```

```
magerun sys:info

                              
  Magento System Information  
                              

+---------------------+-----------------------------------------------------------------------+
| name                | value                                                                 |
+---------------------+-----------------------------------------------------------------------+
| Version             | 1.9.3.10                                                              |
| Edition             | Community                                                             |
```

Reply:
Strange .... directory is also name in camelcase ...

If i use `new Mage_Api_Model_Server_Wsi_Handler();` i get a `Mage_Api_Model_Server_WSI_Handler` object.

Waiting for other reviews :)

Reply:
Kept thise, since it has no negative affect.

Reply:
thats probably OS specific, but I also think the change is ok as the name schema should follow the directory naming.